optimism.tokenlist.json

```diff
@@ -1199,11 +1199,33 @@
       "extensions": {
         "optimismBridgeAddress": "0x4200000000000000000000000000000000000010"
       }
+    },
+    {
+      "chainId": 1,
+      "address": "0x8947da500Eb47F82df21143D0C01A29862a8C3c5",
+      "name": "Optimistic Thales Token",
+      "symbol": "OpTHALES",
+      "decimals": 18,
+      "logoURI": "https://ethereum-optimism.github.io/logos/THALES.png",
+      "extensions": {
+        "optimismBridgeAddress": "0x99C9fc46f92E8a1c0deC1b1747d010903E884bE1"
+      }
+    },
+    {
+      "chainId": 10,
+      "address": "0x217D47011b23BB961eB6D93cA9945B7501a5BB11",
+      "name": "Optimistic Thales Thales",
```

Comment:
Please check the name -> should be "Optimistic Thales Token"

Reply:
Should be all good to go now, thanks a ton for the help & patience Elena. This PR was a first for me, hopefully with more to come.